src/test/java/com/coreos/jetcd/EtcdKVTest.java

```diff
@@ -146,6 +168,27 @@ public void testDelete() throws Exception {
     }
   }
 
+  @Test(dependsOnMethods = "testGetWithPrefix")
```

Comment:
not sure if this is a java practice. but i am not a fan of introducing dependencies into tests.